src/components/Network.tsx

```diff
@@ -31,14 +38,27 @@ function NetworkContent() {
         <Box m={2}>
           <TextField
             value={network}
-            onChange={handleChange}
+            onChange={handleNetworkChange}
             label="Executor URL"
             margin="dense"
             error={!isValid}
             helperText={isValid ? "" : "Please enter a valid Executor URL"}
             fullWidth
           />
         </Box>
+        <Box m={2}>
+          <TextField
```

Comment:
Don't you need a <Menu> to wrap the menu items? Not sure how this looks

Reply:
Nope. https://mui.com/material-ui/react-text-field/#select

Reply:
<img width="491" alt="image" src="https://github.com/user-attachments/assets/70398de3-033b-411e-a40c-a25e21e2f5f6" />